components/runtime/ocaml/Makefile

```diff
@@ -67,6 +67,9 @@ include $(WS_MAKE_RULES)/prep.mk
 include $(WS_MAKE_RULES)/configure.mk
 include $(WS_MAKE_RULES)/ips.mk
 
+
+COMPONENT_BUILD_GMAKE_ARGS += -j1
```

Comment:
this should have been BUILD_ARGS=

Reply:
Did you mean COMPONENT_BUILD_ARGS?
Can't find BUILD_ARGS as macro.
I find 
configure.mk:169:               $(GMAKE) $(COMPONENT_BUILD_GMAKE_ARGS) $(COMPONENT_BUILD_ARGS) \
and
../../openindiana/illumos-gcc/Makefile:95:COMPONENT_BUILD_GMAKE_ARGS=   -j8
This was my way to to decide which macro should be used.
Shall I use COMPONENT_BUILD_ARGS instead of COMPONENT_BUILD_GMAKE_ARGS?
The flag is an argument for make not for the Makefile.

Reply:
Use "COMPONENT_BUILD_ARGS=". Jenkins uses COMPONENT_BUILD_ARGS=-j4 to speed up builds, you have to overwrite it.